src/Analysim.Web/ClientApp/src/app/projects/project-file-explorer/modal-edit/modal-edit.component.ts

```diff
@@ -13,93 +13,92 @@ import { AccountService } from 'src/app/services/account.service';
 })
 export class ModalEditComponent implements OnInit {
 
-  @Input() renameModalRef : BsModalRef
-  @Input() blobList : BlobFileItem[]
-  @Input() currentDirectory : string
-  @Input() projectID : number
+  @Input() renameModalRef: BsModalRef
+  @Input() blobList: BlobFileItem[]
+  @Input() currentDirectory: string
+  @Input() projectID: number
   @Output() newFolderEvent = new EventEmitter<BlobFile>()
 
-  currentUserID : number
+  currentUserID: number
 
-  folderForm : FormGroup
-  folderName : FormControl
-  isLoading : boolean
+  folderForm: FormGroup
+  folderName: FormControl
+  isLoading: boolean
 
   constructor(
-    private formBuilder : FormBuilder,
-    private projectService : ProjectService,
-    private accountService : AccountService
-  ) {}
-  
+    private formBuilder: FormBuilder,
+    private projectService: ProjectService,
+    private accountService: AccountService
+  ) { }
```

Comment:
why so many whitespace changes? Please refrain from making changes on files you don't edit.